types/sticky-session/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for sticky-session 1.1
+// Project: https://github.com/indutny/sticky-session
+// Definitions by: Giovanni Barbaro <https://github.com/vanny96>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Server } from "http";
+
+/**
+ * Method to create the sticky session
+ * @param server Server used to create the session
+ * @param port Port where the app will be listening at
+ * @param options Options to customize the sticky session
+ */
+export function listen(
+    server: Server,
+    port: number,
+    options: {
```

Comment:
Please add [`env`](https://github.com/indutny/sticky-session/blob/master/lib/sticky/api.js#L17).

Reply:
I added ```env``` with ```any``` type because this is what the ```cluster.fork(env)``` method uses. If there is any correction to do let me know!

Reply:
`any` here is probably better.